src/transformers/pipelines.py

```diff
@@ -718,6 +732,8 @@ class TextClassificationPipeline(Pipeline):
             on the associated CUDA device id.
     """
 
+    task = "sentiment-analysis"
```

Comment:
Why would it be "sentiment-analysis" here? The pipeline can do any kind of TextClassification.